testsuite/testcases/src/lib.rs

```diff
@@ -212,6 +212,22 @@ impl dyn NetworkLoadTest {
             .build()
             .map_err(|err| anyhow!("Failed to start runtime for transaction emitter. {}", err))?;
 
+        let clients = ctx
+            .swarm()
+            .get_clients_for_peers(&nodes_to_send_load_to, Duration::from_secs(10));
+
+        // Read first
+        for client in &clients {
+            let start = Instant::now();
+            let _v = rt.block_on(client.get_ledger_information())?;
```

Comment:
Can you explain why is this needed?

Reply:
This is just a sanity check that prints the latency of the connection from Forge test runner <---> validators.